packages/graphql-transformer-common/src/definition.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable prefer-arrow/prefer-arrow-functions */
+/* eslint-disable func-style */
```

Comment:
Makes it easier to read in vscode since it suppresses the red squiggly eslint errors that would otherwise decorate the entire file.

Reply:
Meanwhile with Neovim's LSP capabilities I apparently live in blissful ignorance.